Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -517,7 +527,7 @@ def tc_associated_groups(tc, owners, indicator_value, indicator_type):
     ro.set_http_method('GET')
     ro.set_owner(owners)
     if apiBranch is not None:
-        ro.set_request_uri("/v2/indicators/{}/{}/groups".format(apiBranch, indicator_value))
+        ro.set_request_uri("/v2/indicators/{}/{}/groups".format(apiBranch, quote(indicator_value).replace("/", "%2F")))
```

Comment:
URL Encode indicator value to extract associated groups properly (Only effects hosts and URLs)

Reply:
Please refer to https://github.com/demisto/content/pull/12119/files/2e7b7c84ba157893868c71d0e2dd81f89525f4e1#r608899702